model/instance/lifecycle/patch.go

```diff
@@ -200,29 +205,9 @@ func Patch(i *instance.Instance, opts *Options) error {
 		}
 	}
 
-	if !opts.FromCloudery {
-		managerUpdateSettings(i, clouderyChanges)
-	}
-
 	return nil
 }
 
-func managerUpdateSettings(inst *instance.Instance, changes map[string]interface{}) {
-	if inst.UUID == "" || len(changes) == 0 {
```

Comment:
I think we lost this `if len(changes) == 0 { return }` optimization to avoid calling the cloudery when not necessary

Reply:
This is wanted. We have two consecutive writes not protected by any commit system and this can lead to data corruption between the two bases. In case of a corruption calling `cloudery.SaveInstance` would make the data converge and this corruption would be fixed.

So by removing this optimization I improve the data quality. I choose this tradeoff because we don't have perf issues on this call.

Reply:
@sblaisot is it OK for you?

Reply:
without this optimization, how do you stop the cycle stack -> cloudery -> stack -> cloudery -> ... ?

Reply:
and IIRC this was made to avoid one round trip that was too costly when synchronously attibuting an instance to a user

Reply:
The cycle is stopped by the stack : when the cloudery makes a request, a parameter is added to tell to the stack that this change is coming from the cloudery, and the stack no longer makes a request to the cloudery in that case (cf line 181).